glambie/const/regions.py

```diff
@@ -10,8 +10,20 @@ class RGIRegion():
     rgi7_area: float
     area_change: float
 
+    def __str__(self):
+        return str(self.rgi_id) + '; ' + self.name + '; ' + self.long_name
 
-class Regions():
+
+class MetaRegions(type):
+
+    def __str__(cls):
+        string = ''
```

Comment:
Minor one. I think this is a one liner with '\n'.join(cls.regions).
or with list comprehension if you need to force type '\n'.join([str(region) for region in cls.regions]).